build.yaml

```diff
@@ -679,6 +783,7 @@ job_type:
       archive_artifacts: *flocker_artifacts
       coverage_report: true
       clean_repo: true
+      timeout: 30
 
```

Comment:
building the OSX vagrant box takes longer than 30 minutes, so we will set a separate timeout for that job and define all the others to explicitly say 30 minutes